commands/channel_e2e_test.go

```diff
@@ -251,7 +251,7 @@ func (s *MmctlE2ETestSuite) TestUnarchiveChannelsCmdF() {
 		printer.Clean()
 
 		err := unarchiveChannelsCmdF(s.th.Client, &cobra.Command{}, []string{fmt.Sprintf("%s:%s", s.th.BasicTeam.Id, s.th.BasicDeletedChannel.Name)})
-		s.Require().Nil(err)
+		s.Require().NotNil(err)
```

Comment:
Please use `s.Require().Error(err)` for this. It makes the error messages easier to interpret should the test fail at some point.